crates/y-sweet-worker/src/s3_store.rs

```diff
@@ -0,0 +1,133 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use reqwest::{Client, StatusCode};
+use rusty_s3::{Bucket, Credentials, S3Action};
+use std::{cell::OnceCell, time::Duration};
+use time::OffsetDateTime;
+use y_sweet_core::store::Store;
+
+const PRESIGNED_URL_DURATION_SECONDS: u64 = 60 * 60;
+pub struct S3Store {
+    bucket: Bucket,
+    init_bucket: OnceCell<()>,
```

Comment:
We may not need this at all (I commented below about not creating buckets) but isn’t a `OnceCell` of the unit type just a more complicated `bool`?

Reply:
Oh. yea.

Reply:
well need a RefCell now, but other than that yea.